frb_rust/src/ffi/mod.rs

```diff
@@ -0,0 +1,26 @@
+#[cfg(wasm)]
+pub type DartCObject = wasm_bindgen::JsValue;
```

Comment:
Not sure about this naming: When reading "DartCObject" in a piece of code, the reader will immediately think it is a object in Dart and somehow related to C representation. However, it is indeed a JsValue here.

So, maybe we should introduce an extra name (`type TheFancyNewName = JsValue` on web and `type TheFancyNewName = DartCObject` on io)? And then remove all *cross-platform* usage of "DartCObject" but instead use that new extra name.

On the other hand, non-cross-platform usage of DartCObject should still be kept unchanged. That would make code quite clean I guess :)

Reply:
As you have guessed, the main reason for this odd naming is compatibility, but I understand why this can be a contentious point.

Reply:
I see. But with the proposal above, we get:

* Out internal code has no problem - since we can change it freely
* User's code using DartCObject has no problem as long as it is still used in `io`
* User's code using DartCObject *and* used in `web` has problem, but hey he is going to be used in a new platform and why not spend a few minutes renaming it :)

So I guess the proposal above is quite safe